cloud/gcp-functions/cmd/clusters.go

```diff
@@ -0,0 +1,66 @@
+package clusters
+
+import (
+        "fmt"
+        "log"
+        "net/http"
+        "context"
+        "os"
+        "time"
+        "strconv"
+        "math"
+
+        "google.golang.org/api/compute/v1"
+        "google.golang.org/api/container/v1"
+)
+
+func CleanClusters(w http.ResponseWriter, r *http.Request) {
+    fmt.Fprint(w, "Hello, World test!")
+    ctx := context.Background()
+    computeService, err := compute.NewService(ctx)
+    if err != nil {
+        log.Fatal("Error: create service", err)
+    }
+    containerService, errContainer := container.NewService(ctx)
+    if errContainer != nil {
+        log.Fatal("Error: create containerService", errContainer)
+    }
+
+    project := os.Getenv("GCP_DEV_PROJECT")
+    zonesReq := computeService.Zones.List(project)
+    currentTime := time.Now()
+
+    if err := zonesReq.Pages(ctx, func(page *compute.ZoneList) error {
+        for _, zone := range page.Items {
+            clustersReq, err := containerService.Projects.Zones.Clusters.List(project, zone.Name).Do()
+            if err != nil {
+                log.Fatal(err)
+            }
+            for _, cluster := range clustersReq.Clusters {
+                creationTime, err := time.Parse(time.RFC3339, cluster.CreateTime)
+                if err != nil {
+                    log.Fatal(err)
+                }
+
+                deleteClusterAfterHours := cluster.ResourceLabels["delete-cluster-after-hours"]
+
+                if len(deleteClusterAfterHours) != 0 {
+                    if deleteClusterAfterHours, err := strconv.ParseInt(deleteClusterAfterHours, 10, 64); err == nil {
```

Comment:
rather than checking if error is nil, it's better to check if error is not nil and handle the error.

if there's something wrong with parsing the label value to int, we should log the error and continue. this way we can understand the value that causes the problem and also we can eliminate another level of indentation in the loop